core/contracts/Voting.sol

```diff
@@ -471,8 +466,13 @@ contract Voting is Testable, MultiRole, OracleInterface {
                 return (false, 0, "Price was never requested");
             }
 
+            // Grab the resolution voting round to compute the resolved price.
+            VoteInstance storage voteInstance = priceRequest.voteInstances[resolutionVotingRound];
+            (, int pastResolvedPrice) = voteInstance.resultComputation.getResolvedPrice(
```

Comment:
**How this works for my own understanding**
We use `priceRequest.lastVotingRound` to determine whether or not a vote's been resolved. We can do this because we set the value when a price request is made, and then increment it whenever a vote fails to exceed the threshold. In other words, `priceRequest.lastVotingRound < voteTiming.getLastUpdatedRoundId()` must be true after `_updateRound()` was called if the vote resolved. Therefore, we can assume here that the price was resolved.

Reply:
I guess the follow-up question is, can't `getPrice` and `hasPrice` return stale data?